vobject/hcalendar.py

```diff
@@ -12,7 +12,7 @@
 DTSTART:20051005
 DTEND:20051008
 SUMMARY:Web 2.0 Conference
-LOCATION:Argent Hotel\, San Francisco\, CA
+LOCATION:Argent Hotel, San Francisco, CA
```

Comment:
I think the correct fix here is to use a raw string, and preserve the "\," which are correct iCalendar syntax?